src/components/notfound/NotFound.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import * as React from "react";
+import NotFound from "./NotFound";
+import { MemoryRouter } from "react-router";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+
+describe("NotFound component", () => {
+  it("should render NotFound component", () => {
+    const { getByTestId, getByText } = render(<NotFound />);
+
+    expect(getByTestId("404-page")).toBeInTheDocument();
+    expect(getByText("404 - Not Found!")).toBeInTheDocument();
+    expect(getByTestId("404-page-link")).toBeInTheDocument();
+  });
+
+  it("should render home page after clicking Go Home", () => {
+    const history = createMemoryHistory();
+    const pushSpy = jest.spyOn(history, "push");
+    render(
+      <Router history={history}>
+        <NotFound />
+      </Router>
+    );
```

Comment:
I think we could do that same here as well.

Reply:
@RohitKandimalla Yes done. Thanks!